stackslib/src/config/mod.rs

```diff
@@ -2224,7 +2224,7 @@ pub struct MinerConfig {
     /// Amount of time while mining in nakamoto to wait in between mining interim blocks
     ///
     /// Default: `None`
-    /// Deprecated: use `min_time_between_blocks_ms` instead.
+    /// Deprecated: use [`MinerConfig::min_time_between_blocks_ms`] instead.
```

Comment:
```suggestion
    /// Deprecated: use [`miner::min_time_between_blocks_ms`] instead.
```
i think we should use the toml section vs the function name here